packages/server/src/api/controllers/row/internalSearch.js

```diff
@@ -124,7 +149,14 @@ class QueryBuilder {
     // Construct the actual lucene search query string from JSON structure
     if (this.query.string) {
       build(this.query.string, (key, value) => {
-        return value ? `${key}:${luceneEscape(value.toLowerCase())}*` : null
+        if (!value) {
+          return null
+        }
+        value = preprocess(value, {
+          escape: true,
```

Comment:
since all the pre-process stuff is true by default, do we need to specify it again here? By the look of things we don't actually need the options at all cause nothing actually sets anything to false, everything is always doing everything.

Btw love the pre-process thing, was gonna do this myself, nice little improvement!